thirdparty/README.md

```diff
@@ -71,6 +71,26 @@ Applied upstream PR https://github.com/BinomialLLC/basis_universal/pull/344 to
 fix build with our own copy of zstd (patch in `patches`).
 
 
+## breakpad
+
+- Upstream: https://chromium.googlesource.com/breakpad/breakpad/
+- Version: git (605c51ed96ad44b34c457bbca320e74e194c317e, 2021)
```

Comment:
This is now outdated. @Chaosed0 could I get the commit hash you used to update the Breakpad files so this could be changed to be accurate?

Reply:
Yes, here it is: `527331a471207d28d015e210c12fb8bb8b6732fd`